cmd/storageserv/storage/db/dbcopy/config.go

```diff
@@ -72,7 +74,7 @@ func LoadConfig(file string) string {
 	if !cfg.MicroShardsEnabled {
 		tagFile := fmt.Sprintf("%s/microshard_enabled.txt", dbPath)
 		_, err := os.Stat(tagFile)
-		if !os.IsNotExist(err) {
+		if errors.Is(err, fs.ErrNotExist) {
```

Comment:
missing ! in the if stmt.

Reply:
Updated in https://github.com/paypal/junodb/pull/88/commits/945fb8bbd0d5c9a19a3cb2b6ee8e0d7aec0be7b4